src/modality-specific-files/task-events.md

```diff
@@ -98,12 +99,42 @@ In the accompanying JSON sidecar, the `trial_type` column might look as follows:
             "start": "A red square is displayed to indicate starting",
             "stop": "A blue square is displayed to indicate stopping"
         }
+    },
+    "channel": {
+        "Description": "Channel(s) associated with the event",
+        "Delimiter": "|"
+    },
+    "artifact": {
```

Comment:
Perhaps change "artifact" to HED-annotations:
"HED_annotations": {
        "LongName": "Hierarchical Event Descriptors annotations",
        "Description": "EEG interpretation Hierarchical Event Descriptors annotations"
    }